cmd/rm-main.go

```diff
@@ -121,17 +133,34 @@ func checkRmSyntax(ctx *cli.Context) {
 	isForce := ctx.Bool("force")
 	isRecursive := ctx.Bool("recursive")
 	isStdin := ctx.Bool("stdin")
-
+	isDangerous := ctx.Bool("dangerous")
+	isNamespaceRemoval := false
+	for _, url := range ctx.Args() {
+		// check if url represents just the namespace without bucket or object prefix specified
+		// in the url
+		if !isAliasURLDir(url) && strings.Count(url, string(filepath.Separator)) == 0 {
```

Comment:
A valid input `s3/` and unusual input `s3//` wouldn't work here.  We need to take care properly i.e. we shouldn't use `strings.Count()` inside use url to alias conversion.

If mentioned input is not supported, we have to error out properly.

Reply:
you are correct @balamurugana , added additional validation so that user input errors are caught right away.